IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse1.mo

```diff
@@ -0,0 +1,200 @@
+within IDEAS.Examples.Tutorial.DetailedHouse;
+model DetailedHouse1 "First example model containing one zone"
+  extends Modelica.Icons.Example;
+  package Medium = IDEAS.Media.Air "Air medium";
```

Comment:
Suggestion to rename "Medium" to "MediumAir" to clearly distinct from "MediumWater" (which will be used later in the tutorial).